### PR TITLE
add better logging for new-app argument processing

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@csrwng ptal

**Comment by bparees** [comment]
@openshift/devex ptal.

File: pkg/generate/app/sourcelookup.go [hunk 1]
```diff
@@ -74,17 +74,21 @@ func IsPossibleSourceRepository(s string) bool {
 // IsRemoteRepository checks whether the provided string is a remote repository or not
 func IsRemoteRepository(s string) bool {
 	if !s2igit.New(s2iutil.NewFileSystem()).ValidCloneSpecRemoteOnly(s) {
+		glog.V(5).Infof("%s is not a valid remote git clone spec", s)
```
Comment (gabemontero) [inline pkg/generate/app/sourcelookup.go]: Do these bad urls and clone specs warrant errorf or warningf ?

Comment (bparees) [inline pkg/generate/app/sourcelookup.go]: @gabemontero see the issue i assigned you a few days ago for more context on this behavior and the challenges it presents in terms of how to present errors to users.

**Comment by bparees** [comment]
[test]

File: pkg/generate/app/sourcelookup.go (hunk 1 above)
Comment (gabemontero) [inline pkg/generate/app/sourcelookup.go]: OK, thanks for the clarification - sounds good.

Hadn't gotten to the issue you assigned to me *yet* :-) ... my post PTO catch up is almost there.

Comment (bparees) [inline pkg/generate/app/sourcelookup.go]: > Hadn't gotten to the issue you assigned to me yet

no worries, just figured i'd connect the dots.

Comment (gabemontero) [inline pkg/generate/app/sourcelookup.go]: Cool - yep, dot connecting always helps.

**Comment by bparees** [comment]
@openshift/devex bump

**Comment by jim-minter** [comment]
@bparees, lgtm

**Comment by gabemontero** [comment]
@bparees igtm

**Comment by bparees** [comment]
[merge]

**Pull request merged**